cli/command/registry.go

```diff
@@ -178,7 +179,15 @@ func PromptUserForCredentials(ctx context.Context, cli Cli, argUser, argPassword
 			}
 		}()
 
-		argPassword, err = PromptForInput(ctx, cli.In(), cli.Out(), "Password: ")
+		var credentialPrompt strings.Builder
+		credentialPrompt.WriteString(
```

Comment:
We could reuse the `Info ->` printing from image push, it also handles the non-tty vs tty output:

https://github.com/docker/cli/blob/bdd70c1c61ff633a63ca7e90ab26291f217f1176/cli/command/image/push.go#L187-L207

Reply:
hmm, maybe we could merge your PR https://github.com/docker/cli/pull/5744/files#diff-b4864574dd8732ca90419836bdba5372ed10f8107dc49bf811adbfc0a1506592R15-R36

Reply:
The PR is merged